doc/classes/ConvexPolygonShape2D.xml

```diff
@@ -16,7 +16,7 @@
 			<return type="PoolVector2Array">
 			</return>
 			<description>
-				Return a list of points in either clockwise or counter clockwise order, forming a convex polygon.
+				Returns a list of points in either clockwise or counter clockwise order, forming a convex polygon.
```

Comment:
Set/get aren't needed here because of the member variable.

Reply:
Yes, just fixed in case :)